src/main/io/rcdevice.c

```diff
@@ -256,27 +256,25 @@ void runcamDeviceSimulate5KeyOSDCableButtonRelease(runcamDevice_t *device, rcdev
     runcamDeviceSendRequestAndWaitingResp(device, RCDEVICE_PROTOCOL_COMMAND_5KEY_SIMULATION_RELEASE, NULL, 0, 200, 1, NULL, parseFunc);
 }
 
-static rcdeviceResponseParseContext_t* getWaitingResponse(timeUs_t currentTimeUs)
+static rcdeviceResponseParseContext_t* getWaitingResponse(timeUs_t currentTimeMs)
```

Comment:
Now you are using `timeUs_t` to keep time in ms, which is confusing. Use `timeMs_t`.